scripts/publish_assets.sh

```diff
@@ -10,7 +10,7 @@ set -x
 RELEASE_TAG=$1
 RELEASE_ID=$2
 
-IMG="europe-docker.pkg.dev/kyma-project/prod/api-gateway-manager:${RELEASE_TAG}" make generate-manifests
+IMG="europe-docker.pkg.dev/kyma-project/prod/api-gateway-manager:${RELEASE_TAG}" VERSION=$(git describe --tags --abbrev=0) make generate-manifests
```

Comment:
To make it more obvious and easier to understand we should use the existing RELEASE_TAG, instead of running the git command. 
This script gets the release tag as parameter. This means even if we change how to retrieve the release tag, this will still work as expected as it doesn't use a different mechanism to set VERSION.
```suggestion
IMG="europe-docker.pkg.dev/kyma-project/prod/api-gateway-manager:${RELEASE_TAG}" VERSION=$RELEASE_TAG make generate-manifests
```